chapters/overloaded.tex

```diff
@@ -337,15 +347,15 @@ \section{Overloaded Unary Operations}\doublelabel{overloaded-unary-operations}
 \item
   A binary and/or unary operator-class may only contain functions that
   are allowed for this binary and/or unary operator-class; and in case
-  of `-` it is the union of these sets, since it may define both a unary
+  of `\lstinline!-!' it is the union of these sets, since it may define both a unary
```

Comment:
Should be \lstinline!'-'! as the name of the package is '-' including quotes.

Reply:
Fixed.